gvm/protocols/gmpv208/system/feed.py

```diff
@@ -90,3 +90,19 @@ def sync_feed(self) -> Any:
             The response. See :py:meth:`send_command` for details.
         """
         return self._send_xml_command(XmlCommand("sync_feed"))
+
+    def sync_cert(self) -> Any:
+        """Request a synchronization with the CERT feed service
+
+        Returns:
+            The response. See :py:meth:`send_command` for details.
+        """
+        return self._send_xml_command(XmlCommand("sync_cert"))
+
+    def sync_scap(self) -> Any:
+        """Request a synchronization with the SCAP feed service
+
+        Returns:
+            The response. See :py:meth:`send_command` for details.
+        """
+        return self._send_xml_command(XmlCommand("sync_scap"))
```

Comment:
Could you remove the sync methods? they are not used and i don't know if they still work with gvmd.